src/components/Menu.jsx

```diff
@@ -42,23 +44,40 @@ class Menu extends Component {
                         isFullscreen={isFullscreen}
                         title={title}
                         enableOverlay
+                        isPortal={portalNode}
                     >
                         {rest}
                     </DropdownDialog>
                 ));
-            const newDialog = (isMobile && modalNode)
-                        && ReactDOM.createPortal(
-                            <div className={`cq-modal-dropdown ${className || ''} ${open ? 'stxMenuActive' : ''}`}>
-                                <div
-                                    className="cq-menu-overlay"
-                                    onClick={this.onOverlayClick}
-                                >
-                                    {newDropdown}
-                                </div>
-                            </div>,
-                            modalNode,
-                        )
-                    || newDropdown;
+            let newDialog = newDropdown;
+            if (portalNode) {
+                newDialog = ReactDOM.createPortal(
+                    <div className={`smartcharts-${theme}`}>
```

Comment:
@shayan-binary-2 I'd be better to assign it to a variable and use it as the function parameter. It's more readable.